requirements.txt

```diff
@@ -11,6 +11,7 @@ coverage==6.3.2
 cryptography==36.0.2
 distlib==0.3.4
 Django==4.0.4
+django-coverage-plugin==2.0.2
```

Comment:
I probably done all of this coverage with just `coverage` instead of this new plugin. It's what I'm doing with my day job. Then the script would look a lot like the script looked in your black jack repo. But if this is working, then it's probably fine.

Reply:
It looks like it's reporting coverage for your templates. Which is pretty nice. That might be why the plugin is helpful.

Reply:
I set out to do it the way you were doing it for blackjack, but I didn't understand it fully. While researching to understand it, I found this method.